predator/templates/configmap.yaml

```diff
@@ -10,8 +10,12 @@ metadata:
   namespace: {{ .Release.Namespace }}
 data:
   DATABASE_TYPE: {{ .Values.database.type }}
+{{if .Values.database.name}}
   DATABASE_NAME: {{ .Values.database.name }}
+{{end}}
+{{if .Values.database.address}}
   DATABASE_ADDRESS: {{ .Values.database.address }}
+{{end}}
 {{if eq .Values.database.type "SQLITE"}}
```

Comment:
I suggest going here with if-else

database name and database address are required for any db that is not sqlite